docs/compute-engine/benchmark.md

```diff
@@ -0,0 +1,57 @@
+# Benchmark models
+
+Larq Compute Engine provides prebuilt binaries to benchmark [Larq converted models](/compute-engine/api/converter/) on Android phones or AArch64 based systems like the Raspberry Pi.
+
+=== "Raspberry Pi (64-bit)"
+
+    1. Download the prebuilt benchmarking binary from the [latest release](https://github.com/larq/compute-engine/releases/latest):
+       ```
+       wget <TODO-URL> -O lce_benchmark_model
```

Comment:
This URL needs to be updated once we've release 0.4

Reply:
Same applies to the end to end guide (which is WIP)